internal/internal_workflow.go

```diff
@@ -751,7 +759,7 @@ func (c *channelImpl) Receive(ctx Context, valuePtr interface{}) (more bool) {
 				}
 				break // Corrupt signal. Drop and reset process.
 			}
-			state.yield(fmt.Sprintf("blocked on %s.Receive", c.name))
+			state.yield(printfString{fmt: "blocked on %s.Receive", arg1: c.name})
```

Comment:
```suggestion
			state.yield("blocked on "+c.name+".Receive")
```

Can you microbenchmark this approach?

Reply:
Ah, yeah, that provides a basically identical improvement, and is simpler. I'll switch to that. I might have hoped the compiler could optimize this but I guess Go doesn't go for stuff like that

Reply:
Are you running with PGO during your benchmark?

Reply:
No, I don't think I've ever built a server with PGO. That's an interesting angle to try. I couldn't find any indication that the compiler would partially-evaluate a fmt.Sprintf even with PGO though.

(In general we don't care much about cpu usage since the server is generally io-bound, but in this situation it was replaying a ton of schedule workflows and going over its cpu reservation, so I looked for some quick improvements there.)